commands/store.go

```diff
@@ -31,8 +43,63 @@ var storeCmd = &cobra.Command{
 	Long:  "Allows browsing and deploying OpenFaaS functions from a store",
 }
 
-func storeFindFunction(functionName string, storeItems []schema.StoreItem) *schema.StoreItem {
-	var item schema.StoreItem
+func storeList(store string) ([]schema.StoreFunction, error) {
+
+	var storeData schema.StoreV2
+
+	store = strings.TrimRight(store, "/")
+
+	timeout := 60 * time.Second
+	tlsInsecure := false
+
+	client := proxy.MakeHTTPClient(&timeout, tlsInsecure)
+
+	res, err := client.Get(store)
+	if err != nil {
+		return nil, fmt.Errorf("cannot connect to OpenFaaS store at URL: %s", store)
+	}
+
+	if res.Body != nil {
+		defer res.Body.Close()
+	}
+
+	switch res.StatusCode {
+	case http.StatusOK:
+		bytesOut, err := ioutil.ReadAll(res.Body)
+		if err != nil {
+			return nil, fmt.Errorf("cannot read result from OpenFaaS store at URL: %s", store)
+		}
+
+		jsonErr := json.Unmarshal(bytesOut, &storeData)
+		if jsonErr != nil {
+			return nil, fmt.Errorf("cannot parse result from OpenFaaS store at URL: %s\n%s", store, jsonErr.Error())
+		}
+	default:
+		bytesOut, err := ioutil.ReadAll(res.Body)
+		if err == nil {
+			return nil, fmt.Errorf("server returned unexpected status code: %d - %s", res.StatusCode, string(bytesOut))
+		}
+	}
+
+	return storeData.Functions, nil
+}
+
+func filterStoreList(functions []schema.StoreFunction, platform string) []schema.StoreFunction {
+	var filteredList []schema.StoreFunction
+
+	for _, function := range functions {
+		_, ok := function.Images[platform]
```

Comment:
This needs to be updated to not be case sensitive. Perhaps force the `platform` input to lower case?
Or, change this function to loop and compare with `strings.EqualFold()`...

Reply:
@viveksyngh ?

Reply:
Yes this been taken care while comparing the value and it is updated in the test results.